docs/Token-Swap.md

```diff
@@ -0,0 +1,9 @@
+# Scenario: Token swap
```

Comment:
What is the purpose of this being on its own document? If we want a document with many possible scenarios perhaps name this document `Scenarios.MD` and add more?